.github/dependabot.yml

```diff
@@ -5,11 +5,6 @@ updates:
   # Package ecosystem for yarn is npm
   - package-ecosystem: npm
     directory: /nusight2
-    commit-message:
-      # include: "scope" tells it to add the names of the
-      # upgraded deps to the name, after the prefix
-      include: "scope"
```

Comment:
I think you should keep `commit-message` and `include`, scrapping `prefix`, which was the problem field.

Reply:
I removed `prefix` and it broke because of some other issue

Reply:
what was the other issue?